project/Dependencies.scala

```diff
@@ -2,11 +2,29 @@ import sbt._
 import Keys._
 
 object Dependencies {
-  final val slf4jVersion                   = "1.7.25"
+  final val slf4jVersion                   = "1.7.30"
   final val logbackVersion                 = "1.2.3"
-  final val scalacheckVersion              = "1.15.2"
-  final val scalatestVersion               = "3.2.3"
-  final val scalatestPlusScalacheckVersion = "3.2.3.0"
+  final val scalacheckVersion              =
+    Def.map(scalaBinaryVersion) {
+      case "2.11"      => "1.15.2"
+      case _           => "1.15.3"
+    }
+  final val scalatestVersion               =
+    Def.map(scalaBinaryVersion) {
+      case "2.11"      => "3.2.3"
+      case "2.12"      => "3.2.4"
+      case "2.13"      => "3.2.4"
+      case "3.0.0-M3"  => "3.2.3"
+      case "3.0.0-RC1" => "3.2.4"
+    }
+  final val scalatestPlusScalacheckVersion =
+    Def.map(scalaBinaryVersion) {
+      case "2.11"      => "3.2.3.0"
+      case "2.12"      => "3.2.4.0"
+      case "2.13"      => "3.2.4.0"
+      case "3.0.0-M3"  => "3.2.3.0"
+      case "3.0.0-RC1" => "3.2.4.0"
+    }
```

Comment:
This is a mess.  Scala 2.11 got dropped.  Unlike most libraries, only one version of Scala 3 is supported concurrently. 
 There's no scala.js support in the RC1 version.

We might want to consider porting the specs to MUnit, but I'm more concerned with unblocking people.  Maybe we can mop up the dottyJS support before 1.10.

Reply:
Seems like a good idea if it's better supported.

Reply:
It's usually the first one released, and it has the full matrix from 2.11 through 3.0, which exactly matches our needs.  I don't like to chase fashion, but it really does seem like the best fit now.